### DISCUSSION TITLE
Unexpected 404 error

### DISCUSSION THREAD

**Discussion by Sylvain-BROCAS** [miguelgrinberg/microdot#215] [Q&A] [answered]
Hi,
I'm playing with microdot in order to build a small Ascom Alpaca server (based on this project : https://github.com/RunTJoe/MiPyAlpaca) on an ESP32 devkit V1 board using micropython 1.20.

I based my server programs on this project in order to build my device driver, I just upgraded microdot to V2. I let all the standard Device method as "connected" as is.

I can catch requests on `http://IP:port/management/...`. It gives me a valid Json response with correct data. My device is detected with the right parameters. 
However any request sent to `http://IP:port/api/v1/telescope/0/...` fails and returns an "error 404 : Not Found"

I know that ascom drivers are not a very common knowledge, but I think that the problem is not deep into the code. It seems to be more a problem of the server configuration ou packets exchanges.

You can access my files here to check it : https://github.com/Sylvain-BROCAS/Starget
I wrote a question on Stackoverflow with more details about my errors : https://stackoverflow.com/questions/78130601/alpaca-server-error-404-cant-find-why

Thank you for your help, and huge thanks to miguel for this great module ^^

**Comment by miguelgrinberg** [comment, marked as answer]
This is likely a bug that I inadvertently introduce in the 2.0.4 release. Please update your microdot.py to the 2.0.5 release and let me know if that addresses your issue.

**Comment by Sylvain-BROCAS** [comment]
Thank you for your quick reply,
I can't find the version number on github, how can I download the 2.0.5 ?

**Reply by miguelgrinberg** [reply]
It's currently the latest, so you can download the code from the main branch.

**Comment by Sylvain-BROCAS** [comment]
![image](https://github.com/miguelgrinberg/microdot/assets/82153591/12486c61-6efa-4acd-bf15-12da1c683d2d)
My error changed ! 


Thank you !
